### PR TITLE
IV Scanner Results Translation

### PR DESCRIPTION
**Pull request by Adri1**
image

### CODE REVIEW

File: src/locales/de/battle-message-ui-handler.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { SimpleTranslationEntries } from "#app/plugins/i18n";
+
+export const battleMessageUiHandler: SimpleTranslationEntries = {
+  "ivBest": "Best",
```
Comment (CodeTappert) [inline src/locales/de/battle-message-ui-handler.ts:4]: Please add the following for german.

export const battleMessageUiHandler: SimpleTranslationEntries = {
  "ivBest": "Sensationell",
  "ivFantastic": "Fantastisch",
  "ivVeryGood": "Sehr Gut",
  "ivPrettyGood": "Gut",
  "ivDecent": "Nicht Übel",
  "ivNoGood": "Schlecht",
} as const;

Source https://www.pokewiki.de/Individuelle_St%C3%A4rken#Texte_der_Richter-Funktion_(seit_der_siebten_Generation)

**Review by bennybroseph — APPROVED** [review]

**Review by bennybroseph — CHANGES_REQUESTED** [review]
Do these fit? Fantastique is pretty long depending on the abbreviation for stats

**Comment by Adri1** [comment]
> Do these fit? Fantastique is pretty long depending on the abbreviation for stats

Yes it fits, if you take the "Très bon" in the example, basically add three letters to its length and you have the lenght of "Fantastique"

**Comment by bennybroseph** [comment]
Okay, in the future please include screenshots of the most extreme example

**Review by bennybroseph — APPROVED** [review]

**Pull request merged**